scripts/quests/windurst/Acting_in_Good_Faith.lua

```diff
@@ -74,7 +77,7 @@ quest.sections =
 
         [xi.zone.THE_ELDIEME_NECROPOLIS] =
         {
-            ['qm1'] =
+            ['QM1'] =
```

Comment:
this is not an enum. its the actual npc name. revert the name to how it was, please.

Also, remeber to test your changes

Reply:
Got too trigger happy when hunting for qm1 to update 😔